src/guessTheNumber.js

```diff
@@ -1,3 +1,19 @@
 export default function guessTheNumber() {
-  // your code here
```

Comment:
Good job on this code as well! I like how you used the prompt and alert functions to make this work. Just remove the comment on line 2